File: app/src/main/java/fergaral/datetophoto/listeners/ProgressChangedListener.java
package fergaral.datetophoto.listeners;

import android.os.Parcelable;

import java.io.Serializable;

/**
 * Created by Parejúa on 23/03/2015.
 */
public interface ProgressChangedListener extends Serializable {
    void reportTotal(int total);
    void onProgressChanged(int progress);
    void reportEnd(boolean fromActionShare);
}

File: app/src/main/java/fergaral/datetophoto/listeners/ProgressChangedListener.kt
package fergaral.datetophoto.listeners

import android.os.Parcelable

import java.io.Serializable

/**
 * Created by Parejúa on 23/03/2015.
 */
interface ProgressChangedListener : Serializable {
    fun reportTotal(total: Int)
    fun onProgressChanged(progress: Int)
    fun reportEnd(fromActionShare: Boolean)
}
